src/main/webapp/app/quiz-statistics/drag-and-drop-question-statistic/drag-and-drop-question-statistic.component.ts

```diff
@@ -363,6 +367,15 @@ export class DragAndDropQuestionStatisticComponent implements OnInit, OnDestroy,
         }
     }
 
+    resizeImage() {
+        setTimeout(() => {
+            const image = document.querySelector('.drag-and-drop-quizStatistic-picture jhi-secured-image img') as HTMLImageElement;
```

Comment:
As far as I see, this is a child selector. Is it necessary to include `jhi-secured-image` into the cascade?
If somebody would change the angular component selector, this implementation would break without somebody noticing.

Reply:
Good point! I will change that